app/views/layouts/application.html.erb

```diff
@@ -26,15 +26,10 @@
     </section>
   </nav>
 
-  <%- flash.each do |key, value| -%>
-    <div class="flash flash-<%= key %>">
-      <%= value %>
-    </div>
-  <%- end -%>
-
```

Comment:
This should be added back in! Otherwise, your flash messages won't show up unless you add this code into every single page.